docs/source/release_notes/release-notes.rst

```diff
@@ -42,9 +42,10 @@ develop
     *    - Type
          - Change
 
-    *    -
-         -
-
+    *    - |userbreak|
```

Comment:
This seems a little harsh - we should probably use `|deprecated|`.

We should document the params a little more clearly, but let's get this PR in first which will be useful for several customers